svir/apidoc/svir.dialogs.rst

```diff
@@ -20,10 +20,10 @@ svir.dialogs.drive_oq_engine_server_dialog module
     :undoc-members:
     :show-inheritance:
 
-svir.dialogs.load_dmg_by_asset_as_layer_dialog module
+svir.dialogs.load_avg_damages_rlzs_as_layer_dialog module
 -----------------------------------------------------
 
-.. automodule:: svir.dialogs.load_dmg_by_asset_as_layer_dialog
+.. automodule:: svir.dialogs.load_avg_damages-rlzs_as_layer_dialog
```

Comment:
The "-" is not a good character

Reply:
Thanks Michele. The apidoc is currently abandoned anyway, but I will fix it.